src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -654,25 +644,30 @@ SpinelNCPInstance::update_node_type(NodeType new_node_type)
 	}
 }
 
-
 void
 SpinelNCPInstance::update_link_local_address(struct in6_addr *addr)
```

Comment:
Keeping track of them independently is certainly useful, but using them to keep the network address list in sync seems redundant.